docker/Dockerfile_npm

```diff
@@ -0,0 +1,22 @@
+FROM signalk/signalk-server-base:latest
+
+ADD patch patch
+
+RUN npm i -g /patch/signalk-server-*.tgz \
+  && ln -s /usr/lib/node_modules/signalk-server /home/node/signalk
+
+WORKDIR /home/node/signalk
+RUN npm i /patch/signalk-server-admin-ui-*.tgz --save \
```

Comment:
I think this will install all three packages as siblings and kot the way it is supposed to: admin ui as a dependency of server and used from under signalk-server/node_modules/@signalk/server-admin-ui.

https://github.com/SignalK/signalk-server/blob/master/src/serverroutes.ts#L108

and thus the server will not use them.

Also does the starup script work, as the makn entrypoint `signalk-server` is not where expected?

Reply:
I have been using this very same setup to install npm packages to in the docker earlier and it has worked fine so far.
No issues with symbolic linked path and startup script.